goovn/uuid.go

```diff
@@ -9,5 +11,5 @@ func newUUID() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return id.String(), nil
+	return "row" + strings.Replace(id.String(), "-", "_", -1), nil
```

Comment:
The function name is very generic, so it is not appropriate to return a string with prefix "row".

Reply:
I don't understand how this commit is related to ovn-nbctl behavior. Could you explain a little more? I see the function newUUID() is used only to generate an named ID (which doesn't have to be an uuid format) when doing insert operations. The name is useful only during the transaction execution when a new row inserted in the transaction is referenced by another operation in the same transaction.

Reply:
I see that if named uuid have string like uuid - record not inserted.
This also applied to case then the strings contains _ but, i'm check ovn-nbctl request and see that it sends in named uuid - string with row prefix and uuid with _, so i mimic this behavior.

Reply:
for func name i can change it to newRowUUID for example?